_pydev_imps/_pydev_SocketServer.py

```diff
@@ -336,12 +336,12 @@ def handle_error(self, request, client_address):
         The default is to print a traceback and continue.
 
         """
-        print '-'*40
-        print 'Exception happened during processing of request from',
-        print client_address
+        print ('-'*40)
+        print ('Exception happened during processing of request from',)
+        print (client_address)
```

Comment:
```suggestion
        print('-'*40)
        print('Exception happened during processing of request from',)
        print(client_address)
```

Reply:
Surely you wand to replace line 340 with `print('...', end='')`?